lib/docblockr-worker.js

```diff
@@ -803,6 +803,15 @@ module.exports =
         i++;
       }
       return a;
+    }; 
+    
+    DocBlockrAtom.prototype.get_relevant_editor = function() {
+        var editor = atom.workspace.getActiveTextEditor();
+        if (editor !== undefined)
+            return editor;
+        if (this.event && this.event.target)
+            return this.event.target.getModel();
+        return null;
```

Comment:
I am pretty sure events can be triggered on non-editor elements, in that case calling `getModel();` may not work or not return an `Editor`. I done something similar in another package some time ago perhaps that could be helpful:
https://github.com/robhurring/atom-change-case/pull/37/files